pkg/products/observability/reconciler.go

```diff
@@ -327,6 +327,9 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, serverClient k8scl
 	op, err := controllerutil.CreateOrUpdate(ctx, serverClient, oo, func() error {
 		disabled := true
 		oo.Spec = observability.ObservabilitySpec{
+			AlertManagerDefaultName: "rhoam",
```

Comment:
@philbrookes any opinions on what this should be. we are overriding route/deployment/cr names. In oo they are currently getting appended with kafka. rhoam is just a suggestion. we can go with anything but once they are set in this release we can't change them.